README.md

```diff
@@ -40,7 +40,7 @@ buildNodeJs environment: 'dev', { echo 'do the things' }
 Both pipelines rely on a number of conventions being observed within your repo and Jenkins setup:
 1. The Helm chart must have the same name as the repository
 2. The service name in Docker Compose must have the same name as the repository
-3. The application must be deployed to `/home/node/` in a container (for the Node.js pipeline)
+3. The application must be deployed to a standard location (/home/node for node builds, /home/dotnet for .net Core)
```

Comment:
What do you think to highlighting the path via back-ticks? I think it works well here to distinguish it from the text.

nit: A bit of a nit pick, the correct casing on `.net Core` is `.NET Core`.